packages/utils/src/tabbable.ts

```diff
@@ -22,7 +22,7 @@ export function hasFocusWithin(element: Element) {
   return element.contains(document.activeElement)
 }
 
-export function isHTMLElement(element: Element): element is HTMLElement {
+export function isHTMLElement(element: any): element is HTMLElement {
```

Comment:
I'm curious as to why we needed to change the type of `element` to `any`?

Reply:
It's because it is called with a `Focusable` item which might not be an Element.

Reply:
I'm wondering if we could use [Function overloads](https://www.typescriptlang.org/docs/handbook/functions.html#overloads) to solve this, but that's only my own curiosity. 

These changes make a lot of sense because they focus on the needs of the focus API, not the actual type of the object. Don't mind me 😄

Reply:
> I'm guessing a union type of `Focusable | Element` is a bit too awkward to handle?

I had the same thought 😄